utilities/workers/worker_manager.py

```diff
@@ -11,23 +17,105 @@ class WorkerManager:
+        Create identical workers and append them to a workers list.
 
         count: Number of workers.
         target: Function.
-        args: Arguments to function.
+        class_args: Arguments to function.
+        input_queues: Input queues.
+        output_queues: Output queues.
+        controller: Worker controller.
+
+        Returns whether the workers were able to be created and the Worker Manager.
         """
+
+        create_logger_result, worker_manager_logger = logger.Logger.create("worker_manager")
+        if not create_logger_result:
+            print("Error creating worker manager logger")
+            return False, None
+
+        frame = inspect.currentframe()
+        worker_manager_logger.info("worker manager logger initialized", frame)
+
+        if count <= 0:
+            return False, None
+
+        args = WorkerManager.create_worker_arguments(
+            class_args, input_queues, output_queues, controller
+        )
+
+        workers = []
         for _ in range(0, count):
+            result, worker = WorkerManager.create_single_worker(target, args, worker_manager_logger)
+            if not result:
+                frame = inspect.currentframe()
+                worker_manager_logger.error("Failed to create worker", frame)
+                return False, None
+
+            workers.append(worker)
+
+        return True, WorkerManager(
+            workers,
+        )
+
+    @staticmethod
+    def create_worker_arguments(
+        class_args: tuple,
+        input_queues: "list[queue_proxy_wrapper.QueueProxyWrapper]",
+        output_queues: "list[queue_proxy_wrapper.QueueProxyWrapper]",
+        controller: worker_controller.WorkerController,
+    ) -> tuple:
+        """
+        Creates a tuple containing most arguments for a worker.
+
+        class_args: Class arguments.
+        input_queues: Input queues.
+        output_queues: Output queues.
+        controller: Worker controller.
+
+        Returns a tuple with the arguments.
+        """
+        return class_args + tuple(input_queues) + tuple(output_queues) + (controller,)
+
+    @staticmethod
+    def create_single_worker(target: "(...) -> object", args: tuple, worker_manager_logger: logger) -> Tuple[bool, mp.Process]:  # type: ignore
+        """
+        Creates a single worker.
+
+        target: Fuction
+        args: Worker arguments.
+        worker_manager_logger: Logger for the Worker Manager.
+
+        Returns whether a worker was created and the worker.
+        """
+        try:
             worker = mp.Process(target=target, args=args)
-            self.__workers.append(worker)
+        except Exception as e:  # pylint: disable=broad-exception-caught
+            frame = inspect.currentframe()
+            worker_manager_logger.error(f"Exception raised while creating a worker: {e}", frame)
+            return False, None
+
+        return True, worker
+
+    def __init__(
+        self,
+        workers: "list[mp.Process]",
```

Comment:
Make constructor private with class key. See classes under `modules/` for examples.